src/main/java/ui/components/pickers/MilestonePickerState.java

```diff
@@ -72,7 +75,7 @@ public void selectPreviousBestMatchingMilestone() {
     }
 
     private boolean canIncreaseIndex(Optional<Integer> curSelectedIndex, List<PickerMilestone> list) {
-        return curSelectedIndex.isPresent() && curSelectedIndex.get() < list.size() - 1;
+        return !curSelectedIndex.isPresent() || curSelectedIndex.get() < list.size() - 1;
```

Comment:
Hmm, not too sure on how to use `filter` or `flatMap` here? The resulting value would not be a boolean for returning.